api/v1/address.go

```diff
@@ -0,0 +1,46 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+)
+
+type Addresses []Address
+
+type Address struct {
+	Address   string `json:"address"`
+	IsSigning bool   `json:"is_signing"` // true - active dealmaking from this address, false - still counts as an associated wallet
```

Comment:
Is only one address allowed to sign. Does the address have to sign? What about can_sign?

Reply:
nope, there can be multiple with `is_signing` 

the specifics about how/which one specifically is used is out of scope, handled by MPL50 and/or remote signing network